.github/fabricbot.json

```diff
@@ -12,106 +12,99 @@
             "area-System.Security"
           ],
           "mentionees": [
-            "bartonjs",
+            "dotnet/area-system-security",
             "vcsjones",
-            "krwq",
-            "GrabYourPitchForks"
+            "krwq"
```

Comment:
Did you mea  to keep krwq?

Reply:
@krwq has remained subscribed to those notifications, so it was intentional. However, to make this more straightforward, I went ahead and added them to the dotnet/area-system-security team, so we can remove them from here.